# KondoServoFeedbackMsg.msg

component_lib/GenericComponentMsg component_property

uint16 feedback_degree
uint8 current
uint8 temp
bool is_freed
# see the definitions in component_lib.h
